packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2801,12 +2811,25 @@ class InputDecoration {
   /// {@endtemplate}
   final TextStyle? floatingLabelStyle;
 
+  /// Optional widget that appears below the [InputDecorator.child].
+  ///
+  /// If non-null, the [helper] is displayed below the [InputDecorator.child], in
+  /// the same location as [error]. If a non-null [error] value is
+  /// specified then the [helper] is not shown.
+  ///
+  /// Only one of [helper] and [helperText] can be specified.
+  final Widget? helper;
```

Comment:
Consider adding an example as it may be helpful to the user, similar to how the documentation for `label` does it.

Reply:
I've added an example showcasing the basic usage of the `helper` parameter.